docs_src/microservices/device/profile/Ch-DeviceProfile.md

```diff
@@ -1,6 +1,102 @@
 # Device Profile
 
-![image](EdgeX_DeviceProfile.png)
+The device profile describes a type of device within the EdgeX system. Each
+device managed by a device service has an association with a device profile,
+which defines that device type in terms of the operations which it supports.
 
-The Device Services Device Profile block diagram shows the Device
-Profile Object classes.
+For a full list of device profile fields and their required values see the [Device Profile Reference](./Ch-DeviceProfileRef.md)
+
+Identification
+--------------
+
+The profile contains various identification fields. The `Name` field is required and must be unique in an EdgeX deployment. Other fields are optional - they are not used in EdgeX but may be populated for informational purposes:
+
+* Description
+* Manufacturer
+* Model
+* Labels
+
+DeviceResources
+---------------
+
+A deviceResource specifies a sensor value within a device that may be read
+from or written to either individually or as part of a deviceCommand. It has a
+name for identification and a description for informational purposes.
+
+The device service allows access to deviceResources via its `device`
+REST endpoint.
+
+The `Attributes` in a deviceResource are the device-service-specific parameters
```

Comment:
Do we have documentation about the supported/required Attributes for each of the device services we offer?  As this is written, I'm left scratching my head wondering what an Attribute is and how to use it.

Reply:
We required for new device services that the attributes were documented, see for example https://github.com/edgexfoundry/device-bacnet-c/blob/master/docs/deviceresource_specification.txt
We need to check that all of the existing services have this information